scripts/jupyterkernel/README.md

```diff
@@ -0,0 +1,88 @@
+
+# SystemDS Kernel Setup Guide
+
+This README outlines the steps to set up a Jupyter kernel for SystemDS, focusing on building SystemDS from source, preparing a custom repository with specific dependencies, and setting up a Jupyter kernel.
+
+## Prerequisites
+
+- Java JDK 11 or later
+- Maven
+- Git
+- Jupyter Notebook or JupyterLab
+
+## Step 1: Clone and Build SystemDS
+
+Clone the Apache SystemDS repository and build it. This step ensures that the SystemDS JAR is locally available.
+
+[How to install SystemDS?](https://apache.github.io/systemds/site/install.html)
+
+After the build, save the JAR to your local Maven Repo.
+
+```bash
+mvn install:install-file -Dfile=path/to/your/jar-file.jar -DgroupId=org.apache.sds -DartifactId=sds -Dversion=3.2.0 -Dpackaging=jar
+```
+Note: Ensure that any modifications to groupId, artifactId, and version are carefully mirrored in the pom.xml dependencies section of the Kernel. Inconsistencies between these identifiers in your project setup and the pom.xml file can lead to build failures or dependency resolution issues.
+
+## Step 2: Set Up Your Repository
+
+Clone SystemDS kernel.
+
+```bash
+git clone https://github.com/kubieren/SystemDSKernel.git
+cd SystemDSKernel/kernelsds
+```
```

Comment:
Use the code from the main SystemDS repo, which will be available after this PR is merged.